packages/service-worker/src/services/onboarding/handlers/mnemonicOnboardingHandler.ts

```diff
@@ -93,6 +94,7 @@ export class MnemonicOnboardingHandler implements HandlerType {
     await this.accountsService.addPrimaryAccount({
       walletId,
     });
+    addAllAccountsWithHistory({ walletId });
```

Comment:
Is the `this.accountsService.addPrimaryAccount` call needed here? feels like since `addAllAccountsWithHistory` starts from index 0 we are potentially adding one extra account.